scripts/lib/api/mergeClassMembers.ts

```diff
@@ -63,20 +64,33 @@ export async function mergeClassMembers(
           .use(remarkMath)
           .use(() => {
             return async (tree: Root) => {
-              for (const node of tree.children) {
-                await replaceMembersAfterTitle(
-                  tree,
-                  node,
-                  "Attributes",
-                  attributesAndProps,
-                );
-                await replaceMembersAfterTitle(tree, node, "Methods", methods);
-                await replaceMembersAfterTitle(
-                  tree,
-                  node,
-                  "Methods Defined Here",
-                  methods,
-                );
+              // The attribute and method's section can be found under the class component
+              const mdxClassElements = tree.children.filter(
+                (node): node is MdxJsxFlowElement =>
+                  node.type == "mdxJsxFlowElement" && node.name == "Class",
+              );
+
+              for (const tree of mdxClassElements) {
```

Comment:
Probably we can remove this first loop as we only have one `Class` component. I'll test it and if that's the case I'll simplify it :+1: